gemrb/GUIScripts/iwd2/Enemy.py

```diff
@@ -77,8 +77,8 @@ def OpenEnemyWindow(chargen=0):
 	else:
 		RaceWindow = GemRB.LoadWindow (16, "GUIREC")
 		pc = GemRB.GameGetSelectedPCSingle ()
-		Class = GemRB.GetVar ("LUClass") + 1
-		LevelDiff = GemRB.GetVar ("LevelDiff")
+		Class = GemRB.GetVar ("LUClass") or 0 + 1
```

Comment:
Cases like this need parens, otherwise the +1 will be applied only to the error case.
```
>>> None or 0 + 1
1
>>> 2 or 0 + 1
2
```